2022-04-21T17:48:22.1684535Z Running LuaJIT-tests
2022-04-21T17:48:22.1885692Z PANIC: unprotected error in call to Lua API (builtin/utils.symtab.lua:9: module 'utils.avl' not found:
2022-04-21T17:48:22.1886360Z 	no field package.preload['utils.avl']
